src/components/Modal.css

```diff
@@ -0,0 +1,66 @@
+:root {
+  --off-white: #eee;
+  --gray: #909090;
+  --background: rgb(85, 214, 214);
+  --shadow: rgba(0, 0, 0, 0.3);
```

Comment:
Would be better to define CSS variables in another place, so you can use them wherever you want and have it more organized